api_ui/api.yaml

```diff
@@ -394,7 +394,7 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
-  /v1/credentials/{claim-id}:
+  /v1/offer/{claim-id}:
```

Comment:
It is better to name as `credentials/{claim-id}/offer`. But in order not to make request for mobile apps review again, just wait for new updates from their side